website/sidebars.json

```diff
@@ -88,16 +88,23 @@
       },
       "cita/store",
       {
-        "label": "Hacking",
+        "label": "Advanced-use",
```

Comment:
Is it better to use Chinese?

Reply:
https://github.com/cryptape/citahub-docs/blob/master/website/sidebars.json